docs/src/examples/components/Segment/Variations/SegmentExampleInverted.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import { Segment, Text } from '@stardust-ui/react'
+
+const SegmentExampleInvertedShorthand = () => (
+  <div>
+    <Segment color="purple">Colored segment.</Segment>
+    <br />
+    <Segment inverted color="purple">
+      <Text styles={{ color: 'white' }}>Colored inverted segment.</Text>
```

Comment:
For all examples, remove Text component and use plain text.
Update Segment styles to set the appropriate font color.

The examples will then work similar to SUIR:
https://react.semantic-ui.com/elements/segment/#types-segment

We don't yet have "inverted" concepts in our palette for text colors and other colors.  So for now, just hard code it to '#eee' to avoid passing "bodyBackground" to a font color.  We'll resolve these problems in #423.

Reply:
done, will merge